### ISSUE TITLE
Task Notification

### ISSUE THREAD

**Issue by Quorafind** [lynchjames/obsidian-day-planner#18]
If there is an option to allow plugin to send notification in the upper right corner like as follow, it will be great for those who focus on writing or other things.
![image](https://user-images.githubusercontent.com/13215013/98785858-36238c80-2438-11eb-8d39-fcf052f1f9e9.png)
But I don't know if obsidian has allowed this api.

**Comment by lynchjames** [comment]
Do you mean a notification when the current task changes? Something like this?

```
Current Task: Reading articles
Next Task (in 30 mins): BREAK
```

**Comment by Quorafind** [comment]
@lynchjames Yes

**Comment by lynchjames** [comment]
This is now available in [0.4.0](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.4.0).
